Backend/LinkedOut/credentials/views.py

```diff
@@ -130,6 +130,17 @@ class RecruiterViewSet(viewsets.ModelViewSet):
     serializer_class = RecruiterSerializer
     # permission_classes = [permissions.IsAuthenticated]
 
+    @action(detail=True)
+    def get_jobs(self, request, *args, **kwargs):
+        target_recruiter_id = request.query_params['recruiter_id']
+        job_id = Job.objects.filter(recruiter_id=target_recruiter_id).values_list('id', flat=True)
+        jobs = Job.objects.filter(id__in=job_id).order_by('-posting_date')
+        
+        jsonJobs = []
+        for job in jobs:
+            jsonJobs.append(JobSerializer(job).data)
```

Comment:
Instead of looping like this, you can actually do it in one line by doing: 
`jsonJobs = JobSerializer(Job.objects.filter(id__in=job_id).order_by('-posting_date'), many=True)`

Reply:
I tried replacing line for line and it didnt work.